src/v2/Apps/Conversation/Routes/Conversation/index.tsx

```diff
@@ -16,7 +16,7 @@ interface ConversationRouteProps {
 }
 
 const ConstrainedHeightContainer = styled(Box)`
-  height: calc(100vh - 60px);
+  height: calc(100vh - 103px);
```

Comment:
What does the `103px` in this calculation account for? I ask because 103 feels a bit more arbitrary than 60 and it makes wonder what it represents?

Reply:
Its to accommodate he new header. `100px` left a gap